### PR TITLE
PRM Automated Test - LPS-173731

### PR DESCRIPTION
**Pull request by juliosilveira77**
https://issues.liferay.com/browse/LPS-173731

### CODE REVIEW

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:sf

**Review by AfonsoHSantos — CHANGES_REQUESTED** [review]

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 1]
```diff
@@ -225,6 +225,62 @@ definition {
...
+		
+		}
+				
+		task ("And the information of the second MDF will be visible inside the exported file") {
+			PRMMDFRequest.viewExportedMDF(
+				fileContentList = "Campaign Name 2",
+				fileName = "MDF Requests.csv");
+		}
+				
+	}
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase]: Hi @juliosilveira77, apply the same logic to these tasks

**Review by AfonsoHSantos — CHANGES_REQUESTED** [review]

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
giphy

**Comment by AfonsoHSantos** [comment]
Hi @juliosilveira77 , thanks for working on it!
The code looks good to me, so I'm forwarding it!

**Comment by AfonsoHSantos** [comment]
ci:forward

**Pull request closed without merging**